src/Avalonia.Styling/Controls/Metadata/PseudoClassesAttribute.cs

```diff
@@ -5,14 +5,27 @@
 
 namespace Avalonia.Controls.Metadata
 {
+    /// <summary>
+    /// Defines all pseudoclasses by name referenced and implemented by a control.
+    /// </summary>
+    /// <remarks>
+    /// This is currently used for code-completion in certain IDEs.
```

Comment:
FYI: Dev tools are also using this to generate checkboxes for toggling pseudo classes.